### PR TITLE
[qa_automation] Add test module for all 3 acceptance tests

### PR DESCRIPTION
**Pull request by jtzhao**
This test module contains all 3 acceptance tests, created for KOTD(kernel of the day) testing in the future.

### CODE REVIEW

File: tests/qa_automation/acceptance_all.pm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+# G-Summary: All acceptance tests
+# G-Maintainer: Nathan Zhao <jtzhao@suse.com>
```
Comment (mitiao) [inline tests/qa_automation/acceptance_all.pm]: remove prefix 'G-'  :)

**Comment by jtzhao** [comment]
Test result: http://147.2.207.102/tests/671

**Comment by mitiao** [comment]
I have seen 3 individual tests for process_stress, fs_stress and sched_stress which already exist. Looks like you combined these 3 into 1, right? And I would like to know how you would deal the individuals? @SeroSun I think you had splitted them into 3 individuals long time ago.

**Comment by jtzhao** [comment]
@mitiao This 3-in-one test is for KOTD(kernel of the day) testing. Once the kotd kernel is installed, we'll run all these 3 tests. And it won't conflict with existing ones.

**Pull request merged**